src/faq.md

```diff
@@ -297,26 +193,7 @@ For more information, see [Dart Tools].
 ### Q. Can I build an Android app with Dart?
 
 Yes! You can build an Android app that also works on iOS from a single codebase
-using [Flutter]({{site.flutter}}).
-
-### Q. Can I use Dart on App Engine?
-
-Yes! See
-[Dart and Google Cloud Platform]
-for details.
-
-### Q. What CPU architectures does the Dart VM support?
-
-The Dart VM works on IA-32, x64, MIPS, ARMv5TE, ARMv6, ARMv7, and
-ARM64 processors.
-
-### Q. What are some real-world production deployments of Dart?
```

Comment:
Maybe leave this and link to who uses dart, plus point out, for example, the news about AdWords, AdSense or AdMob. Maybe even say that 75%+ Google revenue flows through these apps? 

That _is_, by the way, a very frequently asked question.